colorizer_data/converter.py

```diff
@@ -0,0 +1,389 @@
+import json
+import logging
+import multiprocessing
+import os
+import pathlib
+import time
+from typing import Dict, List, Optional, Sequence, TypedDict, Union
+
+from aicsimageio import AICSImage
+from dataclasses import dataclass
+from pandas import DataFrame
+from pandas.core.groupby.generic import DataFrameGroupBy
+import pandas as pd
+import numpy as np
+
+
+from colorizer_data.types import BackdropMetadata, ColorizerMetadata, FeatureInfo
+from colorizer_data.utils import (
+    INITIAL_INDEX_COLUMN,
+    generate_frame_paths,
+    get_total_objects,
+    remap_segmented_image,
+    sanitize_key_name,
+    sanitize_path_by_platform,
+    scale_image,
+    update_bounding_box_data,
+)
+from colorizer_data.writer import ColorizerDatasetWriter
+
+DEFAULT_OBJECT_ID_COLUMN = "ID"
+DEFAULT_TIMES_COLUMN = "Frame"
+DEFAULT_TRACK_COLUMN = "Track"
+DEFAULT_IMAGE_COLUMN = "File Path"
+DEFAULT_CENTROID_X_COLUMN = "Centroid X"
+DEFAULT_CENTROID_Y_COLUMN = "Centroid Y"
+DEFAULT_OUTLIER_COLUMN = "Outlier"
```

Comment:
Intended for external use. I ended up not using these in the function signature to make it easier for users to tell what the values are through Intellisense/Pylance/etc. 

With constant:
```
convert_colorizer_data(
    ...
    object_id_column=DEFAULT_OBJECT_ID_COLUMN
)
```

Without constant:
```
convert_colorizer_data(
    ...
    object_id_column="ID",
)
```

Reply:
I'd suggest deleting these if they only appear in the defaults of the function signature

Reply:
Deleted. I do worry though about how users will know the right column names to use... I guess we have to just be very certain we will not ever change them.